public/js/index.js

```diff
@@ -294,7 +294,12 @@ async function addLesson() {
 	const isEditView = createLessonContainer
 		.getAttribute('view')
 		.includes('edit-lesson');
-
+	// Regex to match any number of whitespaces in the content form.
+	var regex = /^\<p\>\s+\<\/p\>$/;
+	if (content === "<p><br></p>" || content.match(regex)) {
```

Comment:
You're on the right track! This pretty much works for all use cases but when I add multiple line breaks it still allows me to create a lesson. There are a couple of approaches I can think of to address this:

1. update the regex to catch multiple line breaks
2. use the `DOMParser` and grab the textContent from the parsed HTML to check if any exists

There are probably more approaches so feel free to explore other options. If you're not sure regarding the `DOMParser` you can check the [docs](https://developer.mozilla.org/en-US/docs/Web/API/DOMParser)

Reply:
Ok , so after having a fight with the Regexes,  I noticed that the Quill returns html tags <p></p> for only whitespace cases and <p><br></p> in multiple line breaks. 
So what I did with the new regex is try to get rid of the html tags, then remove the whitespaces and check if the length of the remaining string is still 0, if so, content is empty, no lesson is added. 

I hope I did not mess up anything, still learning to use git and had some trouble with the commits but I think I managed to get the changes through.

Reply:
Good job figuring out the logic for this. Just left you a small comment on the review. Let me know if it doesn't make sense!